beancount-core/src/account_types.rs

```diff
@@ -1,11 +1,35 @@
 /// Allowed account types.
 ///
 /// <https://docs.google.com/document/d/1wAMVrKIA2qtRGmoVDSUBJGmYZSygUaR0uOMW1GV3YE0/edit#heading=h.17ry42rqbuiu>
-#[derive(Clone, Debug, Eq, PartialEq, Hash)]
+#[derive(Clone, Debug, Eq, PartialEq, Hash, Copy)]
 pub enum AccountType {
     Assets,
     Liabilities,
     Equity,
     Income,
     Expenses,
 }
+
+impl AccountType {
+    /// Get the default name for this account type.
+    ///
+    /// # Example
+    /// ```rust
+    /// use beancount_core::AccountType::*;
+    /// assert_eq!(Assets.default_name(), "Assets");
+    /// assert_eq!(Liabilities.default_name(), "Liabilities");
+    /// assert_eq!(Equity.default_name(), "Equity");
+    /// assert_eq!(Income.default_name(), "Income");
+    /// assert_eq!(Expenses.default_name(), "Expenses");
+    /// ```
+    pub fn default_name(&self) -> &'static str {
```

Comment:
Small nit: the word "default" implies there are other "types" of names.  I'm guessing that is false?  If so, would `name` (or something else) work here?

Reply:
Nevermind, I just got to the part of the PR mentioning account name changes via `option`.

Reply:
Still, it turns out that it's useful to have that in the documentation here.